README.md

```diff
@@ -82,7 +82,16 @@ import { APVideoPlayer } from 'react-native-zapp-bridge';
     },
   };
 
-  return <APVideoPlayer {...{ src, maxwidth, style }} />;
+  // Example
+  const onChange = event => {
+    event.persist();
+
+    if (event.nativeEvent.eventName === 'onVideoEnd') {
```

Comment:
Please document all of the possible eventNames available.

Reply:
@pintocarlos This was done here https://github.com/applicaster/React-Native-Zapp-Bridge/pull/46/commits/1b2c495c0d11b66ec8394b29944361d0249ba4a9

Reply:
@Pretzi @pintocarlos I think for now this is fine. Once both iOS and Android have the protocols to start emitting more events then I will comeback and document that here.